src/RenderState.hpp

```diff
@@ -37,14 +37,17 @@ struct Gosu::RenderState
         }
     }
     
-    void apply_alpha_mode() const
+    void apply_blend_mode() const
```

Comment:
alpha_blend_mode confused me a bit the first time I looked through the files because you actually change not only the Alpha-Channel, so I renamed it.

Reply:
Good point. I guess the whole enum should be named `Gosu::BlendMode` with `BM_ADD` etc. as its members... :/